open-sphere-base/core/src/main/java/io/opensphere/core/util/collections/observable/ExpressionHelperBase.java

```diff
@@ -0,0 +1,41 @@
+package io.opensphere.core.util.collections.observable;
+
+import java.util.function.Predicate;
+
+import javafx.beans.WeakListener;
+
+public class ExpressionHelperBase
```

Comment:
D'ya want this to be an abstract class?  And comments?

Reply:
Sorry, this is a mirror of com.sun.javafx.binding.ExpressionHelperBase. I'll comment and make abstract.

Reply:
fixed